src/CompoundV3YieldOracle.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: BUSL 1.1
+pragma solidity 0.8.13;
+
+import "./interfaces/ICompoundV3YieldOracleAdmin.sol";
+import "./interfaces/IComet.sol";
+// import "./utils/Admin.sol";
+
+contract CompoundV3YieldOracle is ICompoundV3YieldOracleAdmin /*, Admin */ {
+    /**
+     * /////////// CONSTANTS ///////////////
+     */
+    address public constant COMET_USDC_ADDRESS = 0xc3d688B66703497DAA19211EEdff47f25384cdc3;
+    address public constant USDC_ADDRESS = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+
+    /**
+     * ///////////// STATE ///////////////
+     */
+
+    // TODO: ERC20 interface
+    mapping(address => IComet) public tokenAddressToComet;
+
+    constructor() {
+        //admin = msg.sender;
+        setCometAddress(USDC_ADDRESS, COMET_USDC_ADDRESS);
+    }
+
+    /**
+     * ////////// IYieldOracle //////////
+     */
+
+    /// @dev compound III / comet is currently deployed/implemented only on USDC
```

Comment:
Is this why?

Reply:
yes, this is why. exposed the setCometAddress also so that when DAI support comes along, we can upgrade.